### PR TITLE
Ios app

### PR DESCRIPTION
**Pull request by c-44h**

### CODE REVIEW

File: ios-app/RoomMe/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                                   print(st)
+                                     result = st
+                                  }
+                               }
+                               if(result.contains("result: true")) {
+                                    self.push = 1
+                               }
+                               else {
+                                   NSLog("logowanie nieudane")
+                               }*/
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/LoginView.swift]: Comments to remove

File: ios-app/RoomMe/NewFlatView.swift [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                   self.sceneNumber = 0
+               }
+           }) {
+               Text("CREATE")
+           }
+           .padding()
+           .frame(width: 100, height: 100)
+            .cornerRadius(40)
+}
+*/
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/NewFlatView.swift]: Comments to remove

File: ios-app/RoomMe/NewFrienfView.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                               tasks.append(BestFriend(description: listName))
+                               tasksGlobal.append(BestFriend(description: listName))
+                           }
+                       }) {
+                           Text("ADD")
+                       }
+                       .padding()
+                       .frame(width: 150, height: 100)
+                        .cornerRadius(40)
+            //print(exists)
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/NewFrienfView.swift]: to remove

File: ios-app/RoomMe/NewHouseworkView.swift [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                                   NSLog(task.description)
+                               }
+                           }
+                       }) {
+                           Text("ADD")
+                       }
+                       .padding()
+                       .frame(width: 150, height: 100)
+                        .cornerRadius(40)
+            //print(exists)
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/NewHouseworkView.swift]: to remove

File: ios-app/RoomMe/NewRoommateView.swift [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                                   NSLog("nie udalo sie dodac najlepszego przyjaciela")
+                               }
+                           }
+                       }) {
+                           Text("ADD")
+                       }
+                       .padding()
+                       .frame(width: 150, height: 100)
+                        .cornerRadius(40)
+            //print(exists)
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/NewRoommateView.swift]: to remove

File: ios-app/RoomMe/RequestSender.swift [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+                throw URLError(.badServerResponse)
+            }
+        } catch let error {
+            print("JSON Parsing Error: \(error.localizedDescription)")
+        }
+        }
+    
+        task.resume()
+        return result*/
+    }*/
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/RequestSender.swift]: comments to remove

File: ios-app/RoomMe/TaskView.swift [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+                                   withAnimation(.easeOut(duration: 0.3)) {
+                                       self.sceneNumber = 1
+                                   }
+                               }) {
+                                   Text("CREATE NEW HOUSEWORK")
+                               }
+                    
+                               .padding()
+                               .frame(width: 150, height: 100)
+                                .cornerRadius(40)*/
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/TaskView.swift]: Comments to remove

File: ios-app/RoomMe/AddListItemsView.swift [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+                   self.sceneNumber = 0
+               }
+           }) {
+               Text("CREATE")
+           }
+           .padding()
+           .frame(width: 100, height: 100)
+            .cornerRadius(40)
+}
+*/
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/AddListItemsView.swift]: Comments to remove

File: ios-app/RoomMe/LittleHomeView.swift [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    @Binding var sceneNumber: Int
+    @State var tasks = [first1]
+    var body: some View {
+        VStack{
+            HStack {
+            List(tasks) {
+                task in TaskRow1(task: task)
+            }}
+        NavigationView {
+            //Spacer()
```
Comment (mateuszbasiak) [inline ios-app/RoomMe/LittleHomeView.swift]: to remove

File: ios-app/RoomMe/RequestSender.swift [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+//
+//  Created by Student2 on 25/05/2022.
+//
+
+import Foundation
+
+struct RequestSender {
+    var address = "http://3694-95-155-99-184.ngrok.io"
+    /*
+    func postRequest(path: String, body: Dictionary<String, Any>) -> String {
```
Comment (pprochot) [inline ios-app/RoomMe/RequestSender.swift]: Pousuwajcie taki zakomentowany kod

**Review by mateuszbasiak — CHANGES_REQUESTED** [review]

**Review by mateuszbasiak — APPROVED** [review]

**Pull request merged**